build/scripts/install_externals.py

```diff
@@ -0,0 +1,207 @@
+import os
```

Comment:
this entire script should be moved to rexpy, now I know this script also downloads rexpy so I'm not sure how we want to solve this.
it's possible a hardcoded rexpy url to download works and we donwload it uncompressed and then call into rexpy to download the rest

Reply:
We will use rexpy for this which will be installed using pip